gwpy/frequencyseries/tests/test_frequencyseries.py

```diff
@@ -228,6 +228,12 @@ def test_to_from_pycbc(self, array):
         a2 = type(array).from_pycbc(array.to_pycbc(copy=False), copy=False)
         assert shares_memory(array.value, a2.value)
 
+        # test conversion when f0 != 0
```

Comment:
It would be nice to separate this into its own test function, so that it is independently asserted (i.e. so that the error handling is checked even when the `test_to_from_pycbc` function fails early.

```suggestion
    @pytest.mark.requires("pycbc")
    def test_to_from_pycbc_nonzero_f0(self, array):
        """Test `FrequencySeries.to_pycbc` conversion when ``f0 != 0``.
        """
```